_data/simple-icons.json

```diff
@@ -190,68 +190,68 @@
         {
             "title": "Adobe After Effects",
             "hex": "9999FF",
-            "source": "https://www.adobe.com/products/aftereffects.html"
+            "source": "https://www.adobe.com/creativecloud.html"
```

Comment:
We can add https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines/ for every Adobe icon as guidelines.

Maybe https://developer.adobe.com/distribute/cc-brand-assets should be used as source for some icons too.